service/src/main/java/bio/terra/catalog/iam/SamService.java

```diff
@@ -83,7 +83,7 @@ public SystemStatusSystems status() {
       var result = new SystemStatusSystems().ok(samStatus.getOk());
       var samSystems = samStatus.getSystems();
       // Populate error message if Sam status is non-ok
-      if (!samStatus.getOk()) {
+      if (!Boolean.TRUE.equals(result.isOk())) {
```

Comment:
This is verbose. Is there no better way to express this condition?

Reply:
I'm pretty sure I tried
```suggestion
      if (result.isOk() == null || !result.isOk()) {
```
but I can try it again. I'm not sure this is much better though.

Reply:
Oh, I could write
```suggestion
      if (Boolean.FALSE.equals(result.isOk())) {
```